lib/core/terraform_config/generator.rb

```diff
@@ -6,86 +6,106 @@ class Generator
 
     def initialize(config:, template:)
       @config = config
-      @template = template
+      @template = template.deep_underscore_keys.deep_symbolize_keys
     end
 
+    # rubocop:disable Metrics/MethodLength
```

Comment:
Nit: This can be moved to the end of line 13. That way we don't have to enable the rule again later, it gets disabled only for the block.